web-registry/src/components/organisms/EcocreditsTable.tsx

```diff
@@ -62,14 +54,19 @@ export const EcocreditsTable: React.FC<
         <Box sx={{ width: ELLIPSIS_COLUMN_WIDTH }}>{'Project'}</Box>,
         <Box
           sx={{
+            display: 'flex',
+            alignItems: 'center',
             minWidth: {
               xs: 'auto',
               sm: '11rem',
               lg: '13rem',
             },
           }}
         >
-          Batch Denom
+          <BreakText sx={{ maxWidth: 76 }}>{'Ecocredit batch ID'}</BreakText>
```

Comment:
Is this intended to turn "batch denom" into "ecocredit batch ID"? why not using the term used on regen-ledger? has there been any user feedback on it not being clear enough?
If we want to change this, this should be done everywhere so we don't end up with inconsistencies which could confuse users even more:
- credit batch page
- buy/sell forms
- basket table
- cancel
- create batch
...

cc/ @erikalogie

Reply:
@blushi I can definitely revert the change to the ecocredit table while we have this discussion. The new retirement table won't be visible too for now so this new naming would not be visible anywhere.

Reply:
sounds good, then we can do the larger renaming in another PR